safe/impact_function/multi_exposure_wrapper.py

```diff
@@ -876,6 +912,13 @@ def run(self):
 
             analysis_layers.append(impact_function.analysis_impacted)
             aggregation_layers.append(impact_function.aggregation_summary)
+            list_of_analysis_path.append(
+                impact_function.analysis_impacted.source())
+
+        set_provenance(
+            self._provenance,
+            provenance_layer_exposure_summary,
```

Comment:
Maybe better to create its own provenance because this one is a list. Look for `multi` in the provenance definition file.